logdna/lib/cloudWatch.js

```diff
@@ -0,0 +1,38 @@
+// External Libraries
+const zlib = require('zlib');
+
+// Parse the GZipped Log Data
+const parseEvents = (event, callback) => {
+    try {
+        return JSON.parse(zlib.unzipSync(Buffer.from(event.awslogs.data, 'base64')));
+    } catch (error) {
+        callback(error);
+    }
+};
+
+// Prepare the Messages and Options
+exports.createLogs = (events, callback) => {
```

Comment:
My question: why don't we create `class CloudWatch` here and have all inside this class instead of having separate methods exported?

Reply:
There is only one exported method. All other methods are used privately.

Reply:
I understand that but what's the benefit of having `exports.createLogs` instead of having all defined as a method of `class CloudWatch`?